search/includes/classes/class-search.php

```diff
@@ -521,6 +521,9 @@ protected function setup_hooks() {
 
 		// Log details of failed requests
 		add_action( 'ep_invalid_response', [ $this, 'log_ep_invalid_response' ], PHP_INT_MAX, 2 );
+
+		// Lock search algorithm to 3.5
+		add_filter( 'ep_search_algorithm_version', [ $this, 'filter__ep_search_algorithm_version' ], 9999 );
```

Comment:
What do you think about keeping it at a default priority?

Reply:
Makes sense.